support-frontend/app/controllers/CreateSubscriptionController.scala

```diff
@@ -91,14 +91,6 @@ class CreateSubscriptionController(
 
       respondToClient(result, createSupportWorkersRequest.product.billingPeriod)
     } else {
-      val productName = createSupportWorkersRequest.product match {
-        case _: Contribution => "Contribution"
-        case _: DigitalPack => "DigitalPack"
-        case _: Paper => "Paper"
-        case _: GuardianWeekly => "GuardianWeekly"
-      }
-      val cloudwatchEvent = AwsCloudWatchMetricSetup.serverSideValidationFailure(stage, productName)
-      AwsCloudWatchMetricPut(AwsCloudWatchMetricPut.client)(cloudwatchEvent)
```

Comment:
this seemed to have a metric there, but no alarm!  And it has triggered tens of times in the last 6 months in PROD, only for Digital subscriptions

Reply:
Why did you choose not to just alarm on this metric?

Reply:
do you mean 1) why did I change the way the metric is triggered or 2) why did I rename the metric?
1) We didn't see any data coming in due to the outage as it was a JSON parse error.  The new metric is for any failure in the route, rather than just specifically our validation of the data content.  This means it should catch any issue here, including JSON parse error, so we would have seen some data coming in during the failure.
2) I renamed it because it specifically said "Validation" but we are covering all "create" errors however they happen.